plugin.py

```diff
@@ -468,7 +475,7 @@ def branch(self):
             print 'aborted'
             return
 
-        pcur = versioning.Db( psycopg2.connect(self.pg_conn_info()) )
+        pcur = Db( psycopg2.connect(self.pg_conn_info()) )
```

Comment:
We are using very few direct connections, I'd rather move those functionalities to versioning.